wpk/generate_wpk_package.sh

```diff
@@ -72,22 +97,25 @@ function build_container() {
 }
 
 
-
 function help() {
   echo
   echo "Usage: ${0} [OPTIONS]"
+  echo "It is required to use -k or -wk, -wc parameters"
   echo
-  echo "    -t,   --target-system <target>              [Required] Select target wpk to build [linux/windows]"
-  echo "    -b,   --branch <branch>                     [Required] Select Git branch or tag e.g. $BRANCH"
-  echo "    -d,   --destination <path>                  [Required] Set the destination path of package."
-  echo "    -k,   --key-dir <arch>                      [Required] Set the WPK key path to sign package."
-  echo "    -a,   --architecture <arch>                 [Optional] Target architecture of the package [x86_64]."
-  echo "    -j,   --jobs <number>                       [Optional] Number of parallel jobs when compiling."
-  echo "    -pd,  --package-directory <directory>       [Required for windows] Package name to pack on wpk."
-  echo "    -p,   --path <path>                         [Optional] Installation path for the package. By default: /var."
-  echo "    -o,   --output <name>                       [Required] Name to the output package."
-  echo "    -c,   --checksum                            [Optional] Generate checksum"
-  echo "    -h,   --help                                Show this help."
+  echo "    -t,   --target-system <target> [Required] Select target wpk to build [linux/windows]"
+  echo "    -b,   --branch <branch>        [Required] Select Git branch or tag e.g. $BRANCH"
+  echo "    -d,   --destination <path>     [Required] Set the destination path of package."
+  echo "    -pn,  --package-name <name>    [Required for windows] Package name to pack on wpk."
+  echo "    -o,   --output <name>          [Required] Name to the output package."
+  echo "    -k,   --key-dir <arch>         [Optional] Set the WPK key path to sign package."
+  echo "    -wk,  --wpk-key                [Optional] AWS Secrets manager Name/ARN to get WPK private key."
+  echo "    -wc,  --wpk-cert               [Optional] AWS secrets manager Name/ARN to get WPK certificate."
+  echo "    -wr,  --wpk-key-region         [Optional] AWS Region where secrets are stored."
```

Comment:
```suggestion
  echo "    --aws-wpk-key                [Optional] AWS Secrets manager Name/ARN to get WPK private key."
  echo "    --aws-wpk-cert               [Optional] AWS secrets manager Name/ARN to get WPK certificate."
  echo "    --aws-wpk-key-region         [Optional] AWS Region where secrets are stored."
```

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/467/commits/1c82879d25119ef9aeb3808e1a4f12747d8edbf0